### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnnyman727**
Remaining questions:

### CODE REVIEW

**Comment by johnnyman727** [comment]
@tcr want to check this over?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+Our RFC process is heavily inspired by the ECMA/TC39 RFC process. The process is as follows:
+
+### 1. Create a repo for the feature
+One or more champions propose a new feature by creating a repo dedicated to that feature
+Example: https://github.com/rwaldron/exponentiation-operator
+
+The feature is free to describe all aspects of the platform that it interacts with and is described as thoroughly as possible in the readme.md of the repo. Using a repo is convenient for adding and removing fellow feature contributors.
+
+### 2. Post about it on the Forums
+We strongly encourage new RFCs to be posted on [the Forums](forums.tessel.io) for the sake of including as many people as possible.
```
Comment (Frijol) [inline README.md:22]: I'm confused. Where do people go if they want to see a centralized list of all open RFCs?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+Additions and modifications to the feature's proposal are contributed as pull requests (again, great historic record here).
+Example: https://github.com/arv/ecmascript-object-observe/pull/12/files
+
+The repo may include reference implementations, tests, API mockups, use cases, diagrams, illustrations, presentations, pdfs, slide decks, etc. The idea is that the "art" is preserved in a single location.
+Example: https://github.com/tc39/Array.prototype.includes/tree/master/reference-implementation
+
+### 4. Update the central repo with progress
+
+Each proposal is presented in 4 stages which are defined in a [publicly accessible process document](https://docs.google.com/document/d/1DWcHGNI6na1ybcdrUR_wLcz0n5rNJzEbQMGd7sXVjRc/edit?usp=sharing)
```
Comment (Frijol) [inline README.md]: Why is "publicly accessible" needed here? Why not just say "which are defined [here]()"?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+
+The feature proposal is added to a central repo (under "Active RFCs") that keeps a public record of the feature's current stage via a PR on this repo.
```
Comment (Frijol) [inline README.md]: Is this the central repo that we're in right now? Like, they're supposed to PR this readme and add a link to the repo? Make this clear & explicit.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
-# rfcs
-The master record of all RFCs in development
+# Tessel RFCs
+
+For feature additions to the Tessel platform that require some aspect of design or discussion, an RFC (Request For Comment) is necessary. This helps the community collaborate to build robust, thoughtful, and effective additions to the platform. Our hope is to keep the process as light as possible while preserving the reliability and forward-looking nature of Tessel.
+
+Some things don't require RFCs! These are things like documentation fixes or additions or simple bug fixes. If you're not sure, open an issue and ask or pose the question on [the Tessel Slack channel](tessel-slack.herokuapp.com).
+
+## Active RFCs
+None yet! Read about the process below and be the first.
```
Comment (Frijol) [inline README.md]: Add a template example

File: README.md [hunk 5]
```diff
@@ -1,2 +1,43 @@
...
+Example: https://github.com/tc39/Array.prototype.includes/tree/master/reference-implementation
+
+### 4. Update the central repo with progress
+
+Each proposal is presented in 4 stages which are defined in a [publicly accessible process document](https://docs.google.com/document/d/1DWcHGNI6na1ybcdrUR_wLcz0n5rNJzEbQMGd7sXVjRc/edit?usp=sharing)
+
+The feature proposal is added to a central repo (under "Active RFCs") that keeps a public record of the feature's current stage via a PR on this repo.
+Example: https://github.com/tc39/ecma262
+
+Each stage of progress is presented to Tessel Team Members by tagging one on your repo. Current Team Members can be found [here](https://github.com/tessel/project/blob/master/COLLABORATORS.md) The feature either advances to the next stage or continues being developed at the current stage, incorporating new feedback.
```
Comment (Frijol) [inline README.md:41]: Nope, [here](https://github.com/tessel/project/blob/master/TEAM.md)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,43 @@
...
+
+### 5. Merging Code
+Once the feature is complete and all the relevant code has been merged into the relevant repos, the central repo is updated by a Team Member to indicate that the feature is merged.
```
Comment (Frijol) [inline README.md]: "the central repo" = this one?

File: README.md (hunk 2 above)
Comment (johnnyman727) [inline README.md]: Yeah, I can remove that. It was a legacy phrase from Rick's description of the process.

File: README.md (hunk 3 above)
Comment (johnnyman727) [inline README.md]: Yeah, I thought I made it clear with `via a PR on this repo` but I can try to make it more explicit.

**Pull request merged**